src/cuter.erl

```diff
@@ -52,16 +52,19 @@ run(M, F, As, Depth, Options) ->
 %% Runs CutEr on multiple units.
 run(Seeds, Options) ->
   State = state_from_options_and_seeds(Options, Seeds),
-  case init(State) of
-    error ->
-      stop(State),
-      [];
-    ok ->
+  ErroneousInputs =
+    try
+      ok = init(State),
```

Comment:
Since we're going down that road, we should change `init/1` to always return `ok`, and otherwise fail with an exception, rather than returning `ok | error`. 

But this is a change for another PR. If you agree, we can put a TODO here for this.

Thanks!

Reply:
If `init/1` always returns `ok`, the matching does not make much sense anymore, so this code will need to be rewritten. Anyway, this is for another PR, as you write.  I will merge this.